src/ErrorCode.ts

```diff
@@ -0,0 +1,94 @@
+import { NativeModules } from 'react-native';
+
+const { AppLovinMAX } = NativeModules;
+
+const {
```

Comment:
If we want to match Unity, we can also include the ad display failed error (`MAAdapterError.errorCodeAdDisplayFailedError`).